extension/fluentbitextension/README.md

```diff
@@ -0,0 +1,62 @@
+# FluentBit Extension
```

Comment:
Would change the title to "FluentBit subprocess Extension"

Reply:
Done.